### PR TITLE
W-12586956 add apigov env type and env id filters 3x bsj

### PR DESCRIPTION
**Pull request by bjoson-mulesoft**

### CODE REVIEW

File: modules/ROOT/pages/_partials/api-governance.adoc [hunk 1]
```diff
@@ -33,13 +33,29 @@ This command evaluates specified filters to determine the rulesets that would be
...
-|--criteria <filtertype:filtervalue>,<filtertype:filtervalue>,...
-|Enables you to apply filters against which to evaluate to identify rulesets. Specify a list of comma-separated filters where each filter has a type and value in the format filterType:filterValue. Available filters include category, tag, status (lifecycle state), and scope (the targeted API types). Example: tag:tag1,category:category1:value,category:category2:value2,scope:async-api,status:stable,status:development,status:deprecated,scope:rest-api,scope:async-api
+|--criteria <filtertype:filtervalue>,...
+|Enables you to apply filters against which to evaluate to identify rulesets. Specify a list of comma-separated filters where each filter has a type and value in the format `filtertype:filtervalue`. 
+
+Available filters include:
+
+* `scope`: API type, `rest-api` or `async-api` 
+* `tag`: Tag defined for APIs in Exchange
+* `category`: Category defined for APIs in Exchange
```
Comment (sandrau) [inline modules/ROOT/pages/_partials/api-governance.adoc:48]: maybe specify that category should be of the form categoryName:value?

File: modules/ROOT/pages/_partials/api-governance.adoc [hunk 2]
```diff
@@ -33,13 +33,29 @@ This command evaluates specified filters to determine the rulesets that would be
...
+* `status`: Lifecycle state of stable, development, or deprecated
+* `env-type`: Environment type of any, production, or sandbox
+* `env-id`: The ID for the environment name in API Manager. You can get this value using *API Manager* > *Manage Environments*. See xref:api-manager::environments-concept.adoc[Reviewing Environment Concepts]. 
++
+If `env-type` or `env-id` is used, the *API Instance* filter is set in the profile, so that only APIs that have instances are filtered. APIs in the development lifecycle do not have instances.
```
Comment (sandrau) [inline modules/ROOT/pages/_partials/api-governance.adoc:53]: APIs in the development lifecycle **CAN NOT** have instances.

File: modules/ROOT/pages/_partials/api-governance.adoc [hunk 3]
```diff
@@ -281,12 +297,28 @@ See <<exchange-asset-identifiers>>.
...
 
 |--criteria <filtertype:filtervalue>,...
-|Enables you to apply filters to select the list of APIs to which the profile rulesets will apply. Specify a list of comma-separated filters where each filter has a type and value in the format filterType:filterValue. Available filters include category, tag, status (lifecycle state), and scope (the targeted API types). Example: tag:tag1,category:category1:value,category:category2:value2,scope:async-api,status:stable,status:development,status:deprecated,scope:rest-api,scope:async-api
+|Enables you to apply filters to select the list of APIs to which the profile rulesets will apply. Specify a list of comma-separated filters where each filter has a type and value in the format `filtertype:filtervalue`. 
+
+Available filters include:
+
+* `scope`: API type, `rest-api` or `async-api` 
+* `tag`: Tag defined for APIs in Exchange
+* `category`: Category defined for APIs in Exchange
```
Comment (sandrau) [inline modules/ROOT/pages/_partials/api-governance.adoc:311]: same as before maybe clarify the format

File: modules/ROOT/pages/_partials/api-governance.adoc [hunk 4]
```diff
@@ -281,12 +297,28 @@ See <<exchange-asset-identifiers>>.
...
+* `status`: Lifecycle state of stable, development, or deprecated
+* `env-type`: Environment type of any, production, or sandbox
+* `env-id`: The ID for the environment name in API Manager. You can get this value using *API Manager* > *Manage Environments*. See xref:api-manager::environments-concept.adoc[Reviewing Environment Concepts]. 
++
+If `env-type` or `env-id` is used, the *API Instance* filter is set in the profile, so that only APIs that have instances are filtered. APIs in the development lifecycle do not have instances.
```
Comment (sandrau) [inline modules/ROOT/pages/_partials/api-governance.adoc:316]: idem (can not)

File: modules/ROOT/pages/_partials/api-governance.adoc [hunk 5]
```diff
@@ -444,7 +476,24 @@ Besides the default options `--help`, `-f/--fields` and `-o/--output` options, t
...
 See <<exchange-asset-identifiers>>.
 |--criteria <filtertype:filtervalue>,...
-|Enables you to apply filters to select the list of APIs to which the profile rulesets will apply. Specify a list of comma-separated filters where each filter has a type and value in the format filterType:filterValue. Available filters include category, tag, status (lifecycle state), and scope (the targeted API types). Example: tag:tag1,category:category1:value,category:category2:value2,scope:async-api,status:stable,status:development,status:deprecated,scope:rest-api,scope:async-api
+|Enables you to apply filters to select the list of APIs to which the profile rulesets will apply. Specify a list of comma-separated filters where each filter has a type and value in the format `filtertype:filtervalue`. 
+
+Available filters include:
+
+* `scope`: API type, `rest-api` or `async-api` 
+* `tag`: Tag defined for APIs in Exchange
+* `category`: Category defined for APIs in Exchange
```
Comment (sandrau) [inline modules/ROOT/pages/_partials/api-governance.adoc:485]: same

File: modules/ROOT/pages/_partials/api-governance.adoc [hunk 6]
```diff
@@ -444,7 +476,24 @@ Besides the default options `--help`, `-f/--fields` and `-o/--output` options, t
...
+* `status`: Lifecycle state of stable, development, or deprecated
+* `env-type`: Environment type of any, production, or sandbox
+* `env-id`: The ID for the environment name in API Manager. You can get this value using *API Manager* > *Manage Environments*. See xref:api-manager::environments-concept.adoc[Reviewing Environment Concepts]. 
++
+If `env-type` or `env-id` is used, the *API Instance* filter is set in the profile, so that only APIs that have instances are filtered. APIs in the development lifecycle do not have instances.
```
Comment (sandrau) [inline modules/ROOT/pages/_partials/api-governance.adoc:490]: same

**Review by sandrau — APPROVED** [review]

**Review by JennyHajee — APPROVED** [review]

**Pull request merged**